tests/systems/ippo_system_evaluation_test.py

```diff
@@ -33,14 +33,5 @@ def test_ippo(
     test_ippo_system_mt: System,
 ) -> None:
     """Full integration test of ippo system."""
-    (trainer_node,) = test_ippo_system_mt._builder.store.program._program._groups[
-        "trainer"
-    ]
-
-    trainer_node.disable_run()
 
     test_ippo_system_mt.launch()
-    trainer_run = trainer_node.create_handle().dereference()
-
-    for _ in range(5):
-        trainer_run.step()
```

Comment:
the step method here is for running the node and not the step function fromthe trainer but we can say it is quite similar however, the issue is that this causing the checks to fail because we didn't disable the other nodes (executor, evaluator...)